docs/pages/versions/unversioned/sdk/admob.md

```diff
@@ -48,34 +48,34 @@ import {
   AdMobBanner,
   AdMobInterstitial,
   PublisherBanner,
-  AdMobRewarded
+  AdMobRewarded,
+  setTestDeviceIDAsync,
 } from 'expo-ads-admob';
 
+// Set global test device ID
+await setTestDeviceIDAsync("EMULATOR");
+
 // Display a banner
 <AdMobBanner
   bannerSize="fullBanner"
   adUnitID="ca-app-pub-3940256099942544/6300978111" // Test ID, Replace with your-admob-unit-id
-  testDeviceID="EMULATOR"
   servePersonalizedAds // true or false
   onDidFailToReceiveAdWithError={this.bannerError} />
 
 // Display a DFP Publisher banner
 <PublisherBanner
   bannerSize="fullBanner"
   adUnitID="ca-app-pub-3940256099942544/6300978111" // Test ID, Replace with your-admob-unit-id
-  testDeviceID="EMULATOR"
   onDidFailToReceiveAdWithError={this.bannerError}
   onAdMobDispatchAppEvent={this.adMobEvent} />
 
 // Display an interstitial
-AdMobInterstitial.setAdUnitID('ca-app-pub-3940256099942544/1033173712'); // Test ID, Replace with your-admob-unit-id
-AdMobInterstitial.setTestDeviceID('EMULATOR');
+await AdMobInterstitial.setAdUnitID('ca-app-pub-3940256099942544/1033173712'); // Test ID, Replace with your-admob-unit-id
```

Comment:
I know this wouldn't be along the main purpose of the PR, but since we're already changing this code, could we also add a `setAdUnitIDAsync` and deprecate the `setAdUnitID`? This would be more along the lines of what a developer may expect from our naming convention.

Reply:
Uhm I don't know, I haven't investigated the API to that extend. We should probably make a new issue out of that